specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/Jobs.json

```diff
@@ -430,9 +450,136 @@
         },
         "x-ms-examples": {
           "Terminate a Container Apps Job": {
-            "$ref": "./examples/Job_Stop.json"
+            "$ref": "./examples/Job_Stop_Execution.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/stop": {
+      "post": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Terminates execution of a running container apps job",
+        "operationId": "Job_terminateMultipleExecutions",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "jobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "jobExecutionName",
+            "in": "body",
+            "description": "List of all job executions that should be stopped.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JobExecutionNamesCollection"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Container Apps Jobs terminated successfully."
```

Comment:
We need to return a list of jobs and their termination status. Some in the list can be successful some may not be.
So something like this [{jobExecutionName: execution name, Status: success},{jobExecutionName: execution name, Status: not fount}]